javascript/javascript2/week1/spiritAnimalNameGenerator.js

```diff
@@ -0,0 +1,68 @@
+const body = document.querySelector("body");
+
+// create a div to write output to
+const output = document.createElement("div");
+//apply few styles
+output.style.color = "green";
+body.appendChild(output);
+
+function getSpiritAnimalName() {
+  const x = document.getElementById("name").value;
```

Comment:
using alphabets(ex: x) for variables is not a good coding practice. Always use meaningful names for variables.

Reply:
replaced x with name